pkgs/development/libraries/opencv/4.x.nix

```diff
@@ -424,6 +424,17 @@ stdenv.mkDerivation {
   ] ++ lib.optionals stdenv.isDarwin [
     "-DWITH_OPENCL=OFF"
     "-DWITH_LAPACK=OFF"
+
+    # Disable unnecessary vendoring that's enabled by default only for Darwin.
+    # Note that the opencvFlag feature flags listed above still take
+    # precedence, so we can safely list everything here.
+    "-DBUILD_ZLIB=ON"
```

Comment:
I'm confused. https://docs.opencv.org/4.x/db/d05/tutorial_config_reference.html, section

> Options naming conventions

Suggests `-DBUILD_XYZ=ON` _enables_ the "3rdparty library" (which is what opencv seem to call bundled libraries)

Reply:
Thanks for catching that, I did indeed flip all the logic :) oops!

To my defense, I have 0 access to a Darwin host to test this :(